addons/mail/static/tests/core/record.test.js

```diff
@@ -1,120 +1,178 @@
-/** @odoo-module alias=@mail/../tests/core/record_tests default=false */
-const test = QUnit.test; // QUnit.test()
+import { describe, afterEach, beforeAll, beforeEach, expect, test } from "@odoo/hoot";
 
-import { BaseStore, Record, makeStore, modelRegistry } from "@mail/core/common/record";
+import { BaseStore, Record, makeStore } from "@mail/core/common/record";
 
 import { registry } from "@web/core/registry";
-import { clearRegistryWithCleanup, makeTestEnv } from "@web/../tests/helpers/mock_env";
-import { assertSteps, step } from "@web/../tests/utils";
 import { markup, reactive, toRaw } from "@odoo/owl";
+import { mockService } from "@web/../tests/web_test_helpers";
+import { Markup } from "@web/../lib/hoot/hoot_utils";
+import { Matchers } from "@web/../lib/hoot/core/expect";
+import { assertSteps, defineMailModels, start as start2, step } from "../mail_test_helpers";
+import { getMockEnv } from "@web/../tests/_framework/env_test_helpers";
 
-const serviceRegistry = registry.category("services");
+describe.current.tags("desktop");
+defineMailModels();
 
-let start;
-QUnit.module("record", {
-    beforeEach() {
-        serviceRegistry.add("store", { start: (env) => makeStore(env) });
-        clearRegistryWithCleanup(modelRegistry);
-        Record.register();
-        ({ Store: class extends BaseStore {} }).Store.register();
-        start = async () => {
-            const env = await makeTestEnv();
-            return env.services.store;
-        };
-    },
+beforeAll(() => {
+    if (!Matchers.registry["toRecEq"]) {
```

Comment:
I think name are inversed according to standard syntax

`expect(thread).toRecEq(otherThread)`
doesn't make a lot of sense
`expect(thread).toEqRecord(otherThread)`
is reading as expected.